db/migrate/20161216141323_add_website_to_office.rb

```diff
@@ -0,0 +1,5 @@
+class AddWebsiteToOffice < ActiveRecord::Migration
+  def change
+    add_column :offices, :website, :string
+  end
```

Comment:
What's the difference between this change and this change? https://github.com/moneyadviceservice/mas-rad_core/pull/155/files#diff-aefcfd3cbb9a6c9dcb50674ed9b6c9fcR1 ?